setup.py

```diff
@@ -20,13 +20,14 @@
       packages=find_packages(),
       scripts=['scripts/haas', 'scripts/create_bridges'],
       install_requires=['SQLAlchemy==0.9.7',
-                        'Werkzeug==0.9.4',
+                        'Werkzeug>=0.9.4,<0.10',
+                        'Flask>=0.10.1,<0.11',
                         'schema==0.3.1',
                         'importlib==1.0.3',
                         'passlib==1.6.2',
                         'pexpect==3.3',
                         'requests==2.4.1',
-                        'pytest==2.6.2',
+                        'pytest>=2.6.2,<3.0',
```

Comment:
Quoting Kristi Nikolla (2016-03-09 13:52:43)

> >                          'schema==0.3.1',
> >                          'importlib==1.0.3',
> >                          'passlib==1.6.2',
> >                          'pexpect==3.3',
> >                          'requests==2.4.1',
> > -                        'pytest==2.6.2',
> > +                        'pytest>=2.6.2,<3.0',
>
>    Curious about why requiring a pytest older than 3.0.

Normally, the first number indicates backwards-incompatible changes to
the API. There's no 3.0 release yet, but if and when it happens, it may
make changes to the API that cause our tests to break, so we limit this
to versions that are expected to be compatible.

This website explains a common set of rules in detail:

http://semver.org/